spec/unit/devtools/flay/scale/flay_report_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'spec_helper'
+require 'flay'
+
+module Devtools
+  module Flay
+    describe Scale, '#flay_report' do
+      subject(:instance) { described_class.new(minimum_mass: 0, files: []) }
+      let(:flay) do
+        instance_double(::Flay, process: nil, analyze: nil, masses: {})
+      end
+
+      before do
+        allow(::Flay).to receive(:new).with(mass: 0).and_return(flay)
+      end
+
+      it do
```

Comment:
If you want to specify something without a descriptive string, lets use a `specify` block that reads nicer than `it do`.